geoapi/tasks/lidar.py

```diff
@@ -47,6 +47,7 @@ def convert_to_potree(self, pointCloudId: int) -> None:
 
     command = [
         "PotreeConverter",
+        "--verbose",
```

Comment:
Without this verbose command, this command fails (segfault).

I didn't notice this as I had an "older" version of the base image `python 3.7` so was very confused about what was going on...  But `python:3.7` was updated recently (~2 weeks) and for some reason our built version of `PotreeConverter` segfaults without `--version` :( .  It is not clear to me at all why that is the case.